### DISCUSSION TITLE
Deprecation Error for Electron 14

### DISCUSSION THREAD

**Discussion by famt69** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#542] [Q&A]
i am facing an issue while Running an application 
(node:30322) Electron: Loading non-context-aware native module in renderer: '/run/user/1000/snap.g-assist/.org.chromium.Chromium.0KCAuv'. This is deprecated, see https://github.com/electron/electron/issues/18397.

**Comment by im-coder-lg** [comment]
Yeah, I have noticed this. Maybe can you try recreating it in the latest version(1.0.0)?
